hr-panel/.env.template

```diff
@@ -1 +1,2 @@
-REACT_APP_AUTH_API_DOMAIN="http://localhost:8000/"
\ No newline at end of file
+REACT_APP_AUTH_API_DOMAIN="http://localhost:8000/"
```

Comment:
Remove the quotes and the trailing slash after 8000.
```diff
- REACT_APP_AUTH_API_DOMAIN="http://localhost:8000/"
+ REACT_APP_AUTH_API_DOMAIN=http://localhost:8000 
```

Reply:
Also add a new variable. Keep the variables sorted. Use VSCode to sort `Select all text> F1 > Sort Ascending`
```diff
+ REACT_APP_APPLICATION_SECRET=0f6b9c33-3293-4bc4-936e-5a6f94fd55ad
```